src/app/api/config/route.ts

```diff
@@ -17,12 +17,10 @@
  * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { headers } from 'next/headers';
 import { NextResponse } from 'next/server';
 import { getAppConfig } from './config';
 
 export async function GET() {
-	headers();
```

Comment:
why is this removed? @demariadaniel

Reply:
Maybe I misunderstand, but is it not unused? `headers()` returns an object but we do nothing with that object; are you calling this to force the route into a dynamic route? 
I hadn't observed any functional difference but could be wrong

Reply:
Yes it's to force it not to be cached. Runtime var injection in this way will mostly like only be one GET request. But I'm unclear on if this caches at any stage during build time, specifically Nextjs building. unlikely there's a case where this route is HIT with no runtime vars, then ran again with cached GET and some runtime vars...but simply having `headers()` is an easy safeguard to "not cache"

Reply:
Thanks. Restored w/ comment.